src/libs/notifications.test.tsx

```diff
@@ -102,6 +102,21 @@ describe('notify', () => {
     screen.getByLabelText('info notification');
   });
 
+  it('renders custom notifications without an icon', () => {
+    // Arrange
+    notify('custom', 'Test notification', {
+      id: 'test-notification',
+      message: 'This is only a test',
+    });
+
+    // Act
+    render(<div>{notificationContent}</div>);
+
+    // Assert
+    screen.getByText('This is only a test');
+    // the label is only for the icon, which is not present for custom notifications
```

Comment:
This doesn't test that there is no icon… you should either check that there is no element present of the type/role you expect, or at least check that the label text is not present (you should be able to use something like queryByLabelText and verify that it returns null).

Reply:
+1. Could use something like:

```ts
const { container } = render(...);
expect(container.querySelector('svg')).toBeNull();
```